src/main/kotlin/no/skatteetaten/aurora/boober/feature/MountFeature.kt

```diff
@@ -249,14 +265,24 @@ fun List<Mount>.podVolumes(appName: String): List<Volume> {
                 MountType.PVC -> persistentVolumeClaim {
                     claimName = volumeName
                 }
+                MountType.PSAT -> {
+                    projected {
```

Comment:
Jeg synes vi burde ha noe kode som sørger for at man hiver en error i boober når dette skjer hvis man er på OCP3. Ihvertfall på sikt. Hvis det kun er interne brukere som skal bruke dette nå er det greit, men skal det i dokumentasjonen og eksterne skal begynne å bruke det bør vi gi en bedre feilmelding en det kubernetes git siden kulturen ikke leser dem så bra. (Noe som egentlig er veldig dumt...)

Reply:
Tilføyde commit som løser dette.